cli/commands/plan.go

```diff
@@ -55,7 +55,9 @@ func (cmd *planHandler) getPlanName() string {
 	if len(cmd.PlanName) > 0 {
 		return cmd.PlanName
 	}
-	return "deploy"
+	// there is no (and should not be) a case where the plan name is requested here where it is not needed.
+	client.PrintMessageAndExit("Must specify a plan name")
```

Comment:
Also given the `Required()` annotations against the 'plan' argument below, is this even reachable anymore? Maybe try it out -- the arg handling library may return its own error before this code is reached.